solutions/silver/usaco-595.mdx

```diff
@@ -161,32 +160,30 @@ By Project Starcoder
 #include <bits/stdc++.h>
 using namespace std;
 
-#define ll long long
-
 int main() {
 	ifstream cin("div7.in");
 	ofstream cout("div7.out");
 
-	ll N;
+	int N;
```

Comment:
this is supposed to match the code in the video, we should leave it as is (or remove it if it's pretty much the same as the implementation earlier in the module)